src/main/scala/nl/knaw/dans/easy/stage/fileitem/FileItemConf.scala

```diff
@@ -53,14 +54,21 @@ class FileItemConf(args: Seq[String]) extends ScallopConf(args) {
     if (!new File(f).isFile) throw new IllegalArgumentException(s"$f is not an existing file")
     new File(f)
   })
+  def userCategory(default: UserCategory.Value) = singleArgConverter({s =>
+    if (s.trim.isEmpty) default else UserCategory.valueOf(s).get
+  })
+  def string(default: String) = singleArgConverter({s =>
```

Comment:
1. Maybe you should give this function a better name. It is not really descriptive of what this function does.
2. In the License Creator project (as well as others) I have something similar like this inner lambda (see `package.scala`). Maybe it might be interesting to use something similar here as well. Then you get something like `s.toOption.getOrElse(default)`.
3. `userCategory(...)` and `string(...)` look fairly similar. Can you abstract over this?